src/docs/product/integrations/integration-platform/ui-components/issue-link.mdx

```diff
@@ -0,0 +1,145 @@
+---
+title: Issue Link
+sidebar_order: 1
+description: "Learn about the issue link UI component."
+---
+
+With an issue linking UI component, you can connect Sentry issues with a task in your project management system. This functionality provides a way to use any project management tool you use or develop.
+
+The issue link component displays with the text "Link &lt;Service&gt; Issue" in the **Issue Details** sidebar:
+
+![Issue linking sidebar showing Shortcut icon.](issue-sidebar-icons.png)
+
+Clicking the link opens a modal allowing the user to create or link the issue to a task in the external service:
+![Sentry modal that creates a Shortcut Story that's linked to a Sentry issue.](../../issue-tracking/shortcut/shortcut_create_story_with_issue.png)
+
+## Issue Link Schema
+
+```json {filename:schema.json}
+{
+  "elements": [
+    {
+      "type": "issue-link",
+      "link": {
+        "uri": <String>,
+        "required_fields": <Array<FormField>>,
+        "optional_fields": <Array<FormField>>
+      },
+      "create": {
+        // Same as "link" schema
+      }
+    }
+  ]
+}
+```
+
+### Attributes of the Link and Create Fields
+
+- `uri` - (Required) The URI to request when the User submits the Link/Create Issue form.
+- `required_fields` - (Required) List of [FormField](/product/integrations/integration-platform/ui-components/formfield) components the User is required to complete.
+- `optional_fields` - (Optional) List of [FormField](/product/integrations/integration-platform/ui-components/formfield) components the User may complete.
+
+
+## Issue Link Request and Response
+
+When a user attempts to create or link an issue, we will send a request to your service based on the provided `uri`.
+
+### Request Format
+
+We send a `POST` request, so all the information is stored in the body.
+
+### Schema
+
+```json {filename:schema.json}
+{
+  "elements": [
+    {
+      "fields": <Object>,
+      "installationId": <String>,
+      "issueId": <String>,
+      "webUrl": <String>,
+      "project": {"slug": <String>, "id": <String>},
+      "actor": {"type": "user", "name": <String>, "id": <String>},
+    }
+  ]
+}
+```
+
+### Attributes
+
+- `fields` - Key/value pairs of the data in the form fields
+- `installationId` - The ID of the installation associated with the request
+- `issueId` - The ID of the issue in Sentry
+- `webUrl` - The URL of the issue in Sentry
+- `project`
+  - `slug` - The slug of the project in Sentry
+  - `id` - The ID of the project in Sentry
+- `actor`
+  - `name` - The name of the user in Sentry
+  - `id` - The ID of user in Sentry
+
+### Response Format
+
+### Schema
```

Comment:
Again, I feel this looks weird on the sidebar, so I vote to remove `Schema` as a heading (since it isn't one) and change Response Format to an h2 `## Response Format`